.backportrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "targetBranchChoices": [
     { "name": "main", "checked": true },
+    "8.12",
     "8.11",
     "8.10",
```

Comment:
I think `8.9` and `8.10` can be removed, as there's no planned minor release.

`pre-8.10-stable` as well? Not 100% sure but I remember this is a label for some SPO features backported to 8.10. Can we remove this as well? cc @elastic/ingestion-team

Reply:
Approved the PR. This can be addressed in a separate PR.